core/divefilter.cpp

```diff
@@ -14,24 +14,26 @@
 #endif
 
 // Set filter status of dive and return whether it has been changed
-bool DiveFilter::setFilterStatus(struct dive *d, bool shown) const
+bool DiveFilter::setFilterStatus(struct dive *d, bool shown,
+				 const std::vector<dive *> selection, std::vector<dive *> &removeFromSelection) const
```

Comment:
you add two new arguments, `selection` and `removeFromSelection`, but from what I can tell, the former is unused?

Reply:
That was an artifact from code-reshuffling. Fixed.